spec/src/main/asciidoc/circuitbreaker.asciidoc

```diff
@@ -59,26 +53,46 @@ public Connection serviceA() {
    return conn;
 }
 ----
-Below are a couple scenarios to demonstrate when the circuit opens for the above code snippet.
 
 * Scenario 1
 ** Request 1 - success
 ** Request 2 - failure
-** Request 3 - failure
+** Request 3 - success
 ** Request 4 - success
-** Request 5 - CircuitBreakerOpenException
+** Request 5 - failure
+** Request 6 - CircuitBreakerOpenException 
 
-The Request 4 will trigger the circuit to open. The Request 5 will hit the exception.
+In this scenario, request 5 will trigger the circuit to open because out of the last four requests (the `requestVolumeThreshold`), two failed
+which reaches the `failureRatio` of `0.5`. Request 6 will therefore hit the `CircuitBreakerOpenException`.
 
 * Scenario 2
 ** Request 1 - success
 ** Request 2 - failure
-** Request 3 - success
+** Request 3 - failure
 ** Request 4 - success
-** Request 5 - failure
-** Request 6 - CircuitBreakerOpenException 
+** Request 5 - CircuitBreakerOpenException
+
+In this scenario, request 4 will cause the circuit to open. Request 5 will hit the `CircuitBreakerOpenException`.
+Note that request 3 does not cause the circuit to open because the rolling window has not yet reached the `requestVolumeThreshold`.
+
+When a method returns a result, the following rules are applied to determine whether the result is a success or a failure:
```

Comment:
I don't have a solution, but the flow of the text is strange here. We have a section describing when and how the circuit opens, and then without any delimitation, we have "When a method returns a result"... I don't know how to say it, perhaps just ignore me :-)

Reply:
Yeah, I agree. I was trying to stick to the same structure as the other annotations but I'm not sure it really works. I'll try adding some headings and see if that makes it flow better.

Reply:
Headings is also the only possible solution that I could come up with.